x-pack/test_serverless/api_integration/test_suites/common/fleet/default_setup.ts

```diff
@@ -8,42 +8,46 @@
 import { FtrProviderContext } from '../../../ftr_provider_context';
 
 const defaultFleetServerHostId = 'default-fleet-server';
-const defaultFleetServerHostUrl = 'https://localhost:8220';
 const defaultElasticsearchOutputId = 'es-default-output';
-const defaultElasticsearchOutputHostUrl = 'https://localhost:9200';
 
 export async function expectDefaultFleetServer({ getService }: FtrProviderContext) {
   const supertest = getService('supertest');
   const retry = getService('retry');
+  let defaultFleetServerHostUrl: string = '';
 
   await retry.waitForWithTimeout('get default fleet server', 30_000, async () => {
     const { body, status } = await supertest.get(
       `/api/fleet/fleet_server_hosts/${defaultFleetServerHostId}`
     );
     if (status === 200 && body.item.host_urls.length > 0) {
+      defaultFleetServerHostUrl = body.item.host_urls;
```

Comment:
nit: I think it will be safier to set `defaultFleetServerHostUrl = body.item.host_urls[0]` and `return defaultFleetServerHostUrl` at the end. This way the variable is always a string and not sometime a string array

Reply:
I'm surprised that worked, as `defaultFleetServerHostUrl` was declared with `string` type.